README.md

```diff
@@ -19,5 +19,13 @@ slng <search string>
 
 slng will return (up to) the first 3 results by default.
 
+
+#### Contributing
+1. Fork it
+2. Fix it
+3. PR it...?
```

Comment:
Change this to something like "If you see an issue or think up a feature, open an issue on github. See our contribution guidelines <link to md>" or something